.infra/kube/ingress-tls.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: tusd
+  namespace: tus
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
We have a custom nginx configuration which contain a few necessary settings: https://github.com/tus/tusd/blob/master/.infra/files/nginx.conf

Reply:
What specific parameters do you mean? Can you be more precise about them please.

Reply:
Well, all of them are required. :) I am not sure which SSL configuration is used in this PR but they are obviously necessary. Also, some settings (https://github.com/tus/tusd/blob/master/.infra/files/nginx.conf#L52-L65) are more tus-specific and therefore needed.